src/components/ui-component/templates/button.html

```diff
@@ -6,5 +6,6 @@
                >
 		<ui-icon ng-show="me.item.icon.length" icon="{{ me.item.icon }}" ng-style="{color:me.item.getColor()}"></ui-icon>
         <span ng-bind-html="me.item.getLabel()"></span>
+	    <md-tooltip md-delay="700" md-direction="bottom" ng-bind-html="me.item.getTooltip()"></md-tooltip>
```

Comment:
does this need to be optional ? (ng-if etc)  Must ensure it doesn't appear if user clears it out to ""

Reply:
Nothing appears is string is empty